source/scenes/DayOf/scenes/Maps/styles.scss

```diff
@@ -0,0 +1,102 @@
+.map-image {
+  width: 80vw;
+  height: auto;
+  padding-bottom: 20px;
+}
+
+.tab-title {
+  color: #ffffff;
+  font-family: Montserrat, Verdana, serif;
+  font-size: 44px;
+  font-weight: bold;
+  margin-bottom: 28px;
+  margin-top: 48px;
+}
+
+.react-tabs__tab-panel,
+.react-tabs__tab {
+  color: #fd6f6d;
+  font-family: Montserrat, Verdana, serif;
+  font-weight: 600;
+  font-size: 18px;
+  padding: 0;
+}
+
+.react-tabs {
+  overflow: scroll;
+  height: 100vh;
+  position: absolute;
+  top: 0;
+  width: 80vw;
+}
+::-webkit-scrollbar {
+  width: 0px; /* remove scrollbar space */
+  background: transparent; /* optional: just make scrollbar invisible */
+}
+
+.react-tabs__tab-panel--selected {
+  clear: both;
+}
+
+.tab-underline {
+  border-bottom: #fd6f6d 5px var(--border-style-bottom);
+  padding-bottom: 5px;
+  width: fit-content;
+  margin: auto;
+}
+
+.react-tabs__tab {
+  border-bottom: none;
+  float: left;
+  list-style: none;
+  box-sizing: border-box;
+  width: calc(80vw / 4);

```

Comment:
You don't need a `calc` here. `80vw / 4` will be calculated by Sass

Reply:
Unnecessary calcs for most cases, left it for `calc(20vw - 5px)` since it overflows with narrow screens